spec.md

```diff
@@ -1448,6 +1476,62 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 |-----------|-----------|-------------|-------------------|
 | Invalid `starting_token` | 10 ABORTED | Indicates that `starting_token` is not valid. | Caller SHOULD start the `ListVolumes` operation again with an empty `starting_token`. |
 
+#### `GetVolume`
+
+**EXPERIMENTAL FEATURE**
+
+A Controller Plugin MUST implement this `GetVolume` RPC call if it has `GET_VOLUME` capability.
+
+A Controller Plugin SHOULD provide a non-empty `volume_health` field in `GetVolumeResponse` if it has `VOLUME_HEALTH` capability.
+
+`GetVolumeResponse` should contain current information of a volume if it exists.
+If the volume does not exist any more, `GetVolume` should return gRPC error code `NOT_FOUND`.
+
+The `VolumeHealth` message in `VolumeStatus` field in `GetVolumeResponse` can indicate additional health information for the volume.
+
+Whether the volume is still attached to a node is already handled by `LIST_VOLUMES_PUBLISHED_NODES` controller capability.
+This capability MAY be used by CO to reconcile - re-attach the volume if the volume is not attached any more.
+
+```protobuf
+message GetVolumeRequest {
+  option (alpha_message) = true;
+
+  // Identity information for a specific volume. This field is
+  // REQUIRED. GetVolume will return with current volume information.
+  string volume_id = 1;
+}
+
+message GetVolumeResponse {
+  option (alpha_message) = true;
+
+  message VolumeStatus{
+    // volume_health shows error conditions reported by the SP.
+    // This field MUST be specified if the
+    // VOLUME_HEALTH controller capability is supported.
+    VolumeHealth volume_health = 1 [(alpha_field) = true];
```

Comment:
this field appears within an embedded message type whose parent type is already annotated as alpha. this seems redundant and as such is not aligned with the proposal in #365 

read: please remove the alpha annotation here, unless there's strong argument in favor of it

Reply:
Removed